package.json

```diff
@@ -13,17 +13,15 @@
   "author": "StevenC",
   "license": "ISC",
```

Comment:
It would have been nice to update this to reflect our MIT license since you were making changes in the `package.json` anyway, but oh well.

@mikejoh12 If you can make an issue to make sure we do this later that would be great 🙂

Reply:
Ok. I will make an issue for the MIT license.